src/Cli/Microsoft.DotNet.Cli.Utils/Extensions/CommandExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli.Utils.Extensions;
+
+#pragma warning disable IDE0065 // Misplaced using directive
+using Command = System.CommandLine.Command;
+#pragma warning restore IDE0065 // Misplaced using directive
```

Comment:
This is bizarre but necessary since declaring the `using` above the namespace means that anything in the namespace scope will override it. Meaning, `System.DotNet.Cli.Utils.Command` will be used even if an alias is declared. So, you put the alias inside the namespace to have it be used properly.

Reply:
TODO: We should just change that `Command` class to not be called `Command` since `CliCommand` from S.CL was changed back to `Command`, which causes situations like this to happen.

Reply:
My TODO above is just a note to myself in the future, not something I should do in this PR.